Neos.Neos/Classes/Utility/BackendAssetsUtility.php

```diff
@@ -51,6 +51,6 @@ public function shouldLoadMinifiedJavascript()
      */
     public function getCssBuiltVersion()
     {
-        return substr(md5_file('resource://Neos.Neos/Public/Styles/Includes-built.css'), 0, 12);
```

Comment:
Was that the only place the file was used? 🤔

Reply:
It is also used in the template that includes it but we will just remove it later. The ui normally use just Lite or Main ;)